src/components/CTA.tsx

```diff
@@ -47,7 +69,10 @@ export function CTA() {
             />
           ))}
         </div>
-        <p>Join {/* FETCH from Redis */} others unlocking Spanish</p>
+        <p>
+          Join {waitlistCount} other{waitlistCount === 1 ? "" : "s"} unlocking
```

Comment:
Thoughts on this approach? That way it handles the scenario where `waitlistCount` is 1 or 0 
in case the API is down:
```suggestion
          Join {waitlistCount} other{waitlistCount < 2 ? "" : "s"} unlocking
```